src/control/server/faults.go

```diff
@@ -63,6 +63,11 @@ var (
 		fmt.Sprintf("%s instance not started or not responding on dRPC", build.DataPlaneName),
 		"retry the operation or check server logs for more details",
 	)
+	FaultPoolInvalidServiceReps = serverFault(
+		code.ServerPoolInvalidServiceReps,
+		fmt.Sprintf("pool service replicas number should be an odd number between 1 and %d", MaxPoolServiceReps),
```

Comment:
To me this isn't clearly talking about the number of service ranks, it might be made more clear to the user by rewording to "number of service ranks" otherwise it might be confused with service replica rank value. Changing the naming to "service ranks" rather than service replicas it also in line with other changes to the naming elsewhere. Renaming the fault name to "FaultPoolInvalidNumServiceRanks" would also improve clarity.

Reply:
We definitely have some inconsistency throughout the code. I've been moving toward "service replicas" in general rather than "service ranks", as it more clearly conveys their role IMO.

For what it's worth, the dmg help for pool create has referred to "service replicas" for a long time now: https://github.com/daos-stack/daos/blob/master/src/control/cmd/dmg/pool.go#L81